docs/modules/components/pages/processors/sql_raw.adoc

```diff
@@ -267,12 +297,63 @@ args_mapping: root = [ meta("user.id") ]
 
 === `exec_only`
 
-Whether the query result should be discarded. When set to `true` the message contents will remain unchanged, which is useful in cases where you are executing inserts, updates, etc.
+Whether the query result should be discarded. When set to `true` the message contents will remain unchanged, which is useful in cases where you are executing inserts, updates, etc. By default this is true for the last query, and previous queries don't change the results. If set to true for any query but the last one, the subsequent `args_mappings` input is overwritten.
+
+
+*Type*: `bool`
+
+
+=== `queries`
```

Comment:
I am open to different designs/semantics.

@ligfx would love your thoughts feedback on this (there is an example above)

Reply:
Looks good to me 👍 This is pretty much the design I imagined when you said you wanted to have each query as a separate item.

Reply:
That is a good sign - really appreciate you taking a look!